tk.deat.Jazz2Resurrection.yml

```diff
@@ -0,0 +1,43 @@
+app-id: tk.deat.Jazz2Resurrection
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: jazz2
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=all
+
+modules:
+  - name: libopenmpt
+    config-opts:
+      - --disable-examples
+      - --disable-openmpt123
+      - --disable-static
+      - --disable-tests
+      - --without-portaudio
+      - --without-portaudiocpp
+    cleanup:
+      - /include
+      - /lib/*.la
+      - /lib/*.so
```

Comment:
```suggestion
      - "*.la"
      - "*.so"
```

Reply:
FYI I cribbed these lines from other repos that use openmpt, might be worth modifying them too.